src/calendar-list/index.tsx

```diff
@@ -129,7 +131,8 @@ class CalendarList extends Component<Props, State> {
     scrollsToTop: false,
     scrollEnabled: true,
     removeClippedSubviews: Platform.OS === 'android',
-    keyExtractor: (_: any, index: number) => String(index)
+    keyExtractor: (_: any, index: number) => String(index),
+    nestedScrollEnabled: true
```

Comment:
Why the default is true? Since it's a new prop, it should be false

Reply:
> Why the default is true? Since it's a new prop, it should be false

The default value is true because it set to true in the ScrollView from react-native too
https://reactnative.dev/docs/scrollview#nestedscrollenabled-android